rock_gazebo.orogen

```diff
@@ -49,10 +49,17 @@ task_context "ModelTask" do
     # The covariance matrix to be used as velocity covariance. Default is unset (NaN)
     property 'cov_velocity', '/base/Matrix3d'
 
+    # The acceleration of the model w.r.t. the world frame
+    output_port 'ang_accel_samples', '/base/samples/RigidBodyAcceleration'
+    # The covariance matrix to be used as acceleration covariance. Default is unset (NaN)
+    property 'cov_acceleration', '/base/Matrix3d'
+
     property "exported_links", "std/vector<rock_gazebo/LinkExport>"
 
     # The model's pose if one wants to "warp"
     input_port 'model_pose', "/base/samples/RigidBodyState"
+    # Apply wrench to the model
+    input_port 'wrenches', "/base/samples/Wrenches"
```

Comment:
What's the goal with the wrench port. I mean, what do you use it for ?

There is already a way to have per-link interface ... I am wondering if it would not make more sense to have an associated per-link wrench command port, which would avoid the complex name-based resolution. Given that it would not make a lot of sense generally speaking to apply force/torque to more than one link, I feel that it might be a better solution).

Would your use-case fit that ?